app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,9 @@
+<% provide(:title, t("static_pages.about.title")) %>
+  <h1><%= t "static_pages.about.content_about" %></h1>
+  <p>
+    <%= t "static_pages.about.content_about1"  %>
+    <%= link_to t("static_pages.home.content_about2"), "http://www.railstutorial.org/" %>
+      <%= t "static_pages.about.content_about2"  %>
```

Comment:
[IMO] This line is redundant because we already have link above. And we need to remove `</a>` below